src/entry.js

```diff
@@ -46,7 +46,7 @@ module.exports = (argv) => {
   // create the context, logs thread through this
   const context = utils.createContext({command, args, argOpts});
 
-  if (argOpts.version || argOpts.v) {
+  if (command === 'help' && (argOpts.version || argOpts.v)) {
```

Comment:
Can you please confirm this for me again? `zapier --version` is working for me, since `help` becomes the default command if none is specified (look about 4-6 lines above).

I definitely agree `zapier --version` should work, as it's standard, like `zapier -v`.